buildSrc/subprojects/buildquality/src/main/kotlin/gradlebuild/task-properties-validation.gradle.kts

```diff
@@ -22,18 +22,7 @@ plugins {
 val validateTaskName = "validatePlugins"
 val reportFileName = "task-properties/report.txt"
 
-val validateTask = tasks.register<ValidatePlugins>(validateTaskName)
-validateTask {
-    configureValidateTask()
-}
-tasks.named("codeQuality") {
-    dependsOn(validateTask)
-}
-tasks.withType<Test>().configureEach {
-    shouldRunAfter(validateTask)
-}
-
-fun ValidatePlugins.configureValidateTask() {
+tasks.register<ValidatePlugins>(validateTaskName) {
```

Comment:
What about the `test.shouldRunAfter(validateTask)`? Did you remove it on purpose?

Reply:
I removed in on purpose. Because we already define this on the level of the "codeQuality" lifecycle task and I figured it is sufficient to have it there in favour of having it more compact here:

https://github.com/gradle/gradle/blob/154f996c3e089c663cc977b9a315c46954e810dd/gradle/shared-with-buildSrc/code-quality-configuration.gradle.kts#L89-L91

Reply:
Ah, I missed that configuration.